docs/source/license.rst

```diff
@@ -22,4 +28,4 @@ Please note that MAME is a registered trademark of Gregory Ember, and permission
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-Please see `LICENSE.md <https://github.com/mamedev/mame/blob/master/LICENSE.md>`_ for further details.
+Please see the `license information <https://github.com/mamedev/mame#license>`_ for further details.
```

Comment:
This then points you to COPYING for more details, creating a breadcrumb chain.  Perhaps it’s better to just point straight to COPYING? https://github.com/mamedev/mame/blob/master/COPYING

Also, this page: https://docs.mamedev.org/license.html points to LICENSE.md, which no longer exists.

Reply:
Good point. I'll fix that now.